protocol/peersyncing/peersyncing.go

```diff
@@ -25,12 +25,12 @@ func (p *PeerSyncing) AvailableMessages() ([]SyncMessage, error) {
 	return p.persistence.All()
 }
 
-func (p *PeerSyncing) AvailableMessagesByGroupID(groupID []byte, limit int) ([]SyncMessage, error) {
-	return p.persistence.ByGroupID(groupID, limit)
+func (p *PeerSyncing) AvailableMessagesByChatID(groupID []byte, limit int) ([]SyncMessage, error) {
```

Comment:
Can this method and associated persistence-layer fn be removed in favor in `ByChatIDs`?